spec/factories/users_factory.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :user do
+
+    sequence :email { |n| "email#{n}@mail.com"}
```

Comment:
The correct syntax for this is `sequence(:email) { |n| ... }`. This is why your CircleCI is failing 😅